test/stablex/stablecoin_converter.js

```diff
@@ -304,7 +304,7 @@ contract("StablecoinConverter", async accounts => {
 
       await truffleAssert.reverts(
         stablecoinConverter.submitSolution(batchIndex, 0, [], [], [], [], []),
-        "Claimed objective is not more than current solution"
```

Comment:
You could have improved this test case:
"rejects if claimed objective is not better than current"
by checking that improvements smaller than 1% do not go through, but improvements bigger than 1% do.

Reply:
Yes! Great idea. Will do!

Reply:
Unfortunately, 1% better than zero doesn't exit.